wiki/Game_modifier/Nightcore/ko.md

```diff
@@ -0,0 +1,48 @@
+---
+태그:
+  - nightcore
+  - NC
+  - mod
+  - game modifier
+---
+
+# 나이트코어 (모드)
+
+![NC mod icon](/wiki/shared/mods/NC.png "나이트코어 (NC) 모드 아이콘")
+
+*모든 모드의 목록을 확인하려면 [여기](/wiki/Game_modifier)를 눌러주세요.*\
+*[나이트코어 (장르)](https://en.wikipedia.org/wiki/Nightcore "위키피디아")와 혼동하지 말아주세요.*\
+*비슷한 모드: [더블 타임 (모드)](/wiki/Game_modifier/Double_Time)*
+
+## 나이트코어 모드
+
+- 축약형: NC
+- 종류: 난이도 상승 모드
+- 점수 배수:
+  - ![][o!s]: 1.12x
+  - ![][o!t]: 1.12x
+  - ![][o!c]: 1.06x
+  - ![][o!m]: 1.00x
+- 기본 단축키: `D` `D` 또는 `Shift`+`D`
+- 설명: `꺄아아아아아아아아!!`
+- NC를 사용할 수 있는 osu!의 장르: ![][o!s] ![][o!t] ![][o!c] ![][o!m]
+- 같은 계열의 모드: [더블 타임](/wiki/Game_modifier/Double_Time)
+
+## 상세 설명
+
+[**나이트코어**](/wiki/Game_modifier)는 같은 계열의 모드인 더블 타임과 정확히 동일하게 작동하는 모드입니다.
+하지만 이 모드는 음의 높이를 높이고 드럼 소리를 백그라운드에 추가하여 전체 4박자에서 첫 번째와 세 번째 박자는 베이스 드럼으로, 두 번째와 네 번째 박자는 박수를 치는 것을 반복합니다.
+
+더블 타임 모드가 활성화 되었을 때와 동일한 [점수 배수](/wiki/Game_modifier/Score_multiplier)를 유지하며, 비트맵 자체는 더블 타임 모드의 효과를 넘어서는 영향을 받지 않습니다.
```

Comment:
```suggestion
더블 타임 모드가 활성화 되었을 때와 동일한 [점수 배수](/wiki/Game_modifier/Score_multiplier)를 유지하며, 비트맵의 패턴 자체는 더블 타임 모드와 동일하게 진행됩니다.
```
비트맵 자체는 더블 타임 모드의 효과를 넘어서는 영향을 받지 않습니다 is hard to comprehend just by looking at it's meaning. Also I comprehended 비트맵 자체는 as beatmap's pattern (notes and sliders) since 비트맵 can have various component in it's words so I crunched it down to 비트맵의 패턴 to clarify.